django/thunderstore/account/forms.py

```diff
@@ -40,13 +54,17 @@ def __init__(self, user: UserType, *args, **kwargs) -> None:
             queryset=ServiceAccount.objects.filter(owner__members__user=user),
         )
 
-    def clean_service_account(self) -> ServiceAccount:
+    def save(self) -> None:
+        if self.errors:
+            raise ValueError("Cannot save form with errors")
```

Comment:
Is this actually necessary/useful to guard against? Asking because I'd expect Django to have this built in if yes